sw/device/lib/dif/dif_aes.c

```diff
@@ -190,34 +173,24 @@ static dif_result_t configure(const dif_aes_t *aes,
   // if (mode_val == kAesModeFieldValNone) {
   //  return kDifError;
   //}
-  aes_key_field_val_t key_len_val = key_to_field(transaction->key_len);
-  if (key_len_val == kAesKeyFieldValInvalid) {
+  aes_key_field_val_t key_len = key_to_field(transaction->key_len);
+  if (key_len == kAesKeyFieldValInvalid) {
```

Comment:
As discussed in person, I think also this conversion from `dif_aes_key_length_t` to `aes_key_field_val_t` could be removed. The same holds for `aes_operation_t` and `aes_operation_field_val_t`. But someone more familiar with DIFs might know why this was introduced in the first place.

Reply:
When doing that, you can also remove the conversion functions `key_to_field` etc. starting at line 111 as well as the references to the issue at lines 123 and 170.

Reply:
Thanks @vogelpi, Done!

Reply:
If you are happy, I can move this PR from Draft.